src/config.c

```diff
@@ -3122,7 +3122,7 @@ standardConfig static_configs[] = {
     createStringConfig("cluster-config-file", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.cluster_configfile, "nodes.conf", NULL, NULL),
     createStringConfig("cluster-announce-hostname", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.cluster_announce_hostname, NULL, isValidAnnouncedHostname, updateClusterHostname),
     createStringConfig("cluster-announce-human-nodename", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.cluster_announce_human_nodename, NULL, isValidAnnouncedNodename, updateClusterHumanNodename),
-    createStringConfig("syslog-ident", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.syslog_ident, "redis", NULL, NULL),
+    createStringConfig("syslog-ident", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.syslog_ident, "placeholderkv", NULL, NULL),
```

Comment:
Is this fine or should we replace it with `server` ?

Reply:
Can you look into what syslog-ident does? This could also be a breaking change.

Reply:
The ident string is the first parameter passed to openlog(3) - its a string that will be prepended to every logged message.  From the man page:

` The  string  pointed to by ident is prepended to every message, and is typically set to the program name.  If ident is NULL, the program name is used.  (POSIX.1-2008 does  not  specify the behavior when ident is NULL.)`

Probably the new name makes most sense here IMO, although it is arguably a subtle breakage for any 3rd party software searching for the string "redis" in system logs.  Depends on what level of compatibility is sought I guess.

Reply:
@hpatro I'm okay changing this, but let's create an issue tracking all of the little decisions we are looking at making.

Reply:
Added to the top comment.

Reply:
Can we use a macro here, like `PROJECT_NAME` instead of `"placeholderkv"` and define it in one place in server.h so it's easy to change later?

Reply:
echoing on the "breakage" concern. It can break the tooling like log parsers.

>Can we use a macro here, like SERVER_NAME instead of "placeholderkv" and define it in one place in server.h so it's easy to change later?

this is a great idea. we can also introduce either conditional compilation or even new config so that at least there is a grace period for the existing users to upgrade their tools (log parsers, for instance). The "compat" shim can be removed in future releases.